dashboard/legacy/middleware/files_api.rb

```diff
@@ -1037,7 +1037,7 @@ def files_put_file(encrypted_channel_id, filename, body)
         case rating
         when :adult, :racy
           # Incrementing abuse score by 15 to differentiate from manually reported projects
-          new_score = project.increment_abuse(encrypted_channel_id, 15)
+          new_score = project.increment_abuse(encrypted_channel_id, 15, true)
```

Comment:
nit: also add a comment here that automatic moderation can be applied to frozen projects